src/cgame/cg_main.cpp

```diff
@@ -1164,6 +1164,87 @@ static void GenerateNavmeshes()
 	cg.loadingNavmesh = false;
 }
 
+static void SetMapInfo( arenaInfo_t arenaInfo )
+{
+	if ( arenaInfo.longName.empty() )
```

Comment:
It seems wrong to skip making the authors string because there is no long name

Reply:
It was a leftover from when I was producing a single “map by author” string, removed.